src/rest-server/src/models/v2/job/k8s.js

```diff
@@ -484,6 +484,8 @@ const generateTaskRole = (
                   'github.com/fuse': 1,
                   'nvidia.com/gpu':
                     config.taskRoles[taskRole].resourcePerInstance.gpu,
+                  'enflame.com/dtu':
+                    config.taskRoles[taskRole].resourcePerInstance.gpu,
```

Comment:
what is this for? could you include a clear and detailed description explaining the reasons for the changes?

Reply:
enflame DTU(Deep Thinking Unit) is the AI process card from [enflame](https://www.enflame-tech.com/).
With help of [enflame k8s device plugin](https://hub.docker.com/r/enflame/k8s-device-plugin), the DTUs in worker nodes can be detected and  allocated by K8S.
This commit adds the 'enflame.com/dtu' as another GPU resource to OpenPAI.